### ISSUE TITLE
exec busybox failed

### ISSUE THREAD

**Issue by dwidge** [lipanski/docker-static-website#27]
Hello not sure if we can use this CMD on 2.3.0?

```
FROM lipanski/docker-static-website:2.3.0
ARG PUBLIC_URL /.
COPY --from=build /tmp/app/httpd.conf .
COPY --from=build /tmp/app/dist/ .$PUBLIC_URL
CMD ["/busybox", "httpd", "-f", "-v", "-p", "80", "-c", "httpd.conf"]
EXPOSE 80
```
```
[FATAL tini (7)] exec /busybox failed: No such file or directory
 ELIFECYCLE  Command failed.
```

**Comment by dwidge** [comment]
add to readme?
```
CMD ["/busybox_HTTPD", "-f", "-v", "-p", "80", "-c", "httpd.conf"]
```

**Comment by lipanski** [comment]
Fixed in https://github.com/lipanski/docker-static-website/commit/295afd8720b305239b981c853cc08edf89a28da7 and released as 2.3.1

The new CMD line is:

```
CMD ["/busybox-httpd", "-f", "-v", "-p", "80", "-c", "httpd.conf"]
```

**Comment by NabilSnigdho** [comment]
@lipanski it would be very nice to update the command in https://lipanski.com/posts/smallest-docker-image-static-website also.
